lib/sb3_schema.json

```diff
@@ -71,15 +71,15 @@
                 "assetId": { "$ref": "#/definitions/assetId"},
                 "dataFormat": {
                     "type": "string",
-                    "enum": ["wav", "WAV", "wave", "WAVE", "mp3", "MP3"]
+                    "enum": ["wav", "wave", "mp3"]
                 },
                 "format": {
                     "type": "string",
                     "enum": ["", "adpcm"]
                 },
                 "md5ext": {
                     "type": "string",
-                    "pattern": "^[a-fA-F0-9]{32}.[a-zA-Z]+$"
+                    "pattern": "^[a-fA-F0-9]{32}.[a-zA-Z0-9]+$"
```

Comment:
Ha! Good catch ;)

Reply:
Sorry to interrupt but shouldn't the dot be escaped, if this is a regex? :-) `\.`

Reply:
good catch, @tjvr!